src/Microsoft.Identity.Client/Logger.cs

```diff
@@ -117,6 +118,7 @@ public sealed class Logger
         /// For details see https://aka.ms/msal-net-logging
         /// </summary>
         /// <exception cref="ArgumentException">will be thrown if the LogCallback was already set</exception>
+        [Obsolete("Logging is now specified per ClientApplication.  See https://aka.ms/msal-net-application-configuration.", true)]
```

Comment:
>Obsolete("Loggi [](start = 9, length = 15)

this stuff will cause failures in the build, however should we also simply throw an exception instead of keeping the code around?